src/main/scala/no/ndla/learningpathapi/model/domain/config/ConfigKey.scala

```diff
@@ -0,0 +1,13 @@
+/*
+ * Part of NDLA draft-api.
+ * Copyright (C) 2019 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.learningpathapi.model.domain.config
+
+object ConfigKey extends Enumeration {
+  val IsExamPeriod: ConfigKey.Value = Value("IS_EXAM_PERIOD")
```

Comment:
Er ikkje veldig glad i navnet IS_EXAM_PERIOD og tilsvarende. Eg veit det er det som er bestillinga, men hadde foretrekt at variabelen var feks "WRITE_RESTRICTED" eller "IS_WRITE_RESTRICTED" for å gjøre det meir generellt. Kom gjerne med forslag.

Reply:
Fikser det! :)